doc/source/installing.rst

```diff
@@ -407,7 +407,7 @@ development version of yt instead of the latest stable release, you will need
   $ git clone https://github.com/yt-project/yt
   $ cd yt
   $ git checkout master
-  $ pip install -e . --user --prefix=
+  $ pip install . --user --prefix=
```

Comment:
I'm not sure offhand whether `--prefix=` can be passed to pip at all. I think you need to do say `--install-option="--prefix="`

I'm also not sure offhand whether `--prefix=` is even necessary anymore, I think it was a workaround for an issue on older fedora installs?